package/Dockerfile-windows.agent

```diff
@@ -22,7 +22,14 @@ RUN pushd c:\; \
     popd;
 RUN $URL = 'https://{0}/fleet/{1}/fleetagent-windows-amd64.exe' -f $env:RELEASES, $env:VERSION; \
     New-Item -Path 'c:\\' -Name 'fleet' -ItemType 'directory'; \
-    Write-Host ('Downloading dapper from {0} ...' -f $URL); \
+    Write-Host ('Downloading fleet-agent from {0} ...' -f $URL); \
     [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; \
-    Invoke-WebRequest -UseBasicParsing -OutFile c:/fleet/fleetagent-windows.exe -Uri $URL;
+    Invoke-RestMethod -UseBasicParsing -OutFile c:/fleet/fleetagent-windows.exe -Uri $URL;
+
+FROM mcr.microsoft.com/windows/nanoserver:${SERVER_VERSION}
+
+RUN mkdir "git" "fleet"
+COPY --from=build c:/git/. c:/git/.
```

Comment:
What does the `git` directory provide us? The binary is statically linked and I cannot think of any runtime dependencies. I could be missing something though.